src/modules/counter.js

```diff
@@ -0,0 +1,7 @@
+const itemCounter = async () => {
+  const response = await fetch('https://www.themealdb.com/api/json/v1/1/search.php?s=');
+  const data = await response.json();
+  return data.meals.length;
+};
+
```

Comment:
However, this entire function depends on the data provided by the base API in other to get the number of meals.
According to the requirements, we are supposed to calculate the number of meals independent of the API data.

Reference the stated issue for more information on it. This will also ensure that your test is successful when implemented.

You can reach out to me if you have any concerns.